apps/ledger-live-mobile/src/components/FabActions/FabButtonBar.tsx

```diff
@@ -15,21 +15,24 @@ function FabButtonBar({ data }: Props) {
       showsHorizontalScrollIndicator={false}
       contentContainerStyle={{ paddingHorizontal: 16 }}
     >
-      {data.map(({ children, onPress, Icon, buttonProps, disabled }, index) => (
-        <Button
-          size={"small"}
-          Icon={Icon}
-          iconPosition={"left"}
-          type={buttonProps?.type ?? "shade"}
-          outline={buttonProps?.outline ?? true}
-          disabled={disabled}
-          onPress={onPress}
-          key={index}
-          mr={3}
-        >
-          {children}
-        </Button>
-      ))}
```

Comment:
We'll need to fix these prettier issues. Make sure your eslint and prettier extensions are installed properly and your save on format is working (google these to give it a go - I'll give you a hand if it's still not working)